lib/core.ps1

```diff
@@ -682,7 +682,7 @@ function shim($path, $global, $name, $arg) {
             ") else (",
             "    powershell -noprofile -ex unrestricted -file `"$resolved_path`" $arg %args%",
             ")"
-        ) -join "`r`n" | Out-File "$shim.cmd" -Encoding ASCII
+        ) -join "`r`n" | Out-UTF8File "$shim.cmd"
```

Comment:
This breaks on CMD, 
![image](https://user-images.githubusercontent.com/5764917/157431775-768bca72-aebb-41fc-9226-907385a54570.png)

Reply:
If the PATH contains special characters, the user might need to enable support for them, like this: https://github.com/kiedtl/winfetch/issues/132#issuecomment-1049901256

Reply:
That will not work on Windows 7.

Reply:
Does this matter?

Reply:
Does Windows 7 lack that setting or does not work on 7 even after enabling that setting? Idk anything about 7.

Reply:
Lack that `beta` feature.

Reply:
Oh...

Reply:
This setting was introduced on Windows 10, so only Windows 10 (build 17134 or above explicitly) can use this setting to turn on the UTF-8 as ASCII feature. Otherwise, users need to use trick like `chcp 65001` to work with UTF-8. https://en.wikipedia.org/wiki/Unicode_in_Microsoft_Windows

This is a well-known issue, and `chcp 65001` has been [talked about widely as a workaround](https://stackoverflow.com/questions/57131654/) to enable UTF-8 support for CMD/PowerShell(old versions), though this trick has [bad user experiences](https://dev.to/mattn/please-stop-hack-chcp-65001-27db).

Reply:
I don't know how many Win  users are out there, but this feature will allow us to close #2573 and #2230 as well

Reply:
This is Win7/CMD bug, not on PowerShell, right? So it's okay and ready to be merged IMO.

Reply:
> Does this matter?

It depends. I'm here just saying it's a breaking change, we have no idea of how many Windows 7 users in our community, I tend to push it forword though.

Reply:
There are certainly some Win 7 users, I mean is it only causes bug on CMD and run well on PowerShell? If so, then it's okay.